### DISCUSSION TITLE
Github actions not cloning the repo

### DISCUSSION THREAD

**Discussion by RpxdYTX** [community/community#37587] [Repositories]
I made a repository with only a CMakeLists.txt file, a header in include/ and 2 C sources in src/.
The lists file only sets the minimum cmake version, the project name alongside version and languages, the build type default to Debug, the compiler flags and the executable with the sources and header.
The header only defines a heap-based growable string, one of the sources implement it and the other one simply tests if it is working.

Making a build action fails with CMake not being able to find the lists file, however, i'm sure that the it's in the right place. Just to try and troubleshoot, i tried running `ls -A` but it returned nothing, using just gcc didn't solve it either.
The commands used on cmake's build were `cmake -B build` and `cmake --build build`, and on gcc's were `gcc -g -Wall -Wextra -I./include -c src/heap_string.c -o build/heap_string.o`, `gcc -g -Wall -Wextra -I./include -c src/main.c -o build/main.o` and `gcc -g -Wall -Wextra -I./include -o build/main build/main.o build/heap_string.o`.
Then i just ran the executable on the build workflow after compiling the program, but both gcc and cmake act like there's no code. Though, i haven't used `cd` or changed `defaults.run`, even with `${{ github.workspace}}` it doesn't work.

So what do I do?

**Comment by SonilNail** [comment]
(◉‿◉) (◉‿◉) (◉‿◉) (◉‿◉) (◉‿◉) (◉‿◉) (◉‿◉) (◉‿◉) (◉‿◉) (◉‿◉) (◉‿◉) (◉‿◉) 
 (✷‿✷) (✷‿✷) (✷‿✷) (✷‿✷) (✷‿✷) (✷‿✷) (✷‿✷) (✷‿✷) (✷‿✷) (✷‿✷) (✷‿✷) (✷‿✷) (✷‿✷) (✷‿✷) (✷‿✷) (✷‿✷) (✷‿✷) (✷‿✷) (✷‿✷) (✷‿✷) (✷‿✷) (✷‿✷) (✷‿✷) (✷‿✷) (✷‿✷) (✷‿✷) 
 ffffffffffffgffffgggfffffffffffffffffffffffffff
 (｡◕‿◕｡) (｡◕‿◕｡) (｡◕‿◕｡) (｡◕‿◕｡) (｡◕‿◕｡) (｡◕‿◕｡) (｡◕‿◕｡) (｡◕‿◕｡) (｡◕‿◕｡) 
     tuuuuuuuuuuuuuuuuuuu
  Mu Youtube  channel    https://youtube.com/channel/UCRGoYNHxQoSPzYuvxGM55ow
  WhatsApp no    0780161145
     Facebook Haji Almas
     ـــــــــــــــــــــــــــــــــــــــــــــــــــــــــــــــــــــــــــــــــــــــــــــــــــــــــــــــــــــــــــــــــــــــــــــــــــــــــــــــــــــــــــــــــــــــــــــــــــــــــــــــــــــــــــــ،،،،،،،،،ــــــــــــــــــــــــــــــــــــــــــــــــــــــــــــــــــــــــــــــــــــــــــــــــــــــــــــــــــ
     your name 
     Subscribe to our channel 
     Facebook mining 20014 
     Facebook mining 2006 
     Facebook extraction 2008,2009 Crack 
     The crack of the country 
     Choose password 123456                           1234567                                                           123456789
     How many numbers to crack? 50000
     https://youtube.com/channel/UCRGoYNHxQoSPzYuvxGM55ow
